cli/dcoscli/node/main.py

```diff
@@ -148,3 +165,61 @@ def _mesos_files(master, slave_id):
         slave = mesos.get_master().slave(slave_id)
         files.append(mesos.MesosFile('/slave/log', slave=slave))
     return files
+
+
+def _ssh(master, slave, option, config_file, user):
+    """SSH into a DCOS node.  Since only the masters are definitely
+    publicly available, we first ssh into an arbitrary master, then
+    hop to the desired node.
+
+    :param master: True if we ssh into the master
```

Comment:
This description is confusing since the above comment says `we always ssh into an arbitrary master`